Makefile

```diff
@@ -172,6 +172,9 @@ build/mgmt-%: $(GO_FILES) $(MCL_FILES) | bindata lang funcgen
 	@echo "Building: $(PROGRAM), os/arch: $*, version: $(SVERSION)..."
 	@time env GOOS=${GOOS} GOARCH=${GOARCH} go build -i -ldflags=$(PKGNAME)="-X main.program=$(PROGRAM) -X main.version=$(SVERSION) ${LDFLAGS}" -o $@ $(BUILD_FLAGS)
 
+goinstall: $(GO_FILES) $(MCL_FILES) | bindata lang funcgen
+	time go install -i -ldflags=$(PKGNAME)="-X main.program=$(PROGRAM) -X main.version=$(SVERSION) ${LDFLAGS}" $(BUILD_FLAGS)
```

Comment:
Don't we want the `env GOOS=${GOOS} GOARCH=${GOARCH}` as with the `go build` ?

Reply:
Hm, I'll review where those come from. To the user it would make most sense for the `go install` command to build with the native host GOOS and GOARCH which are implied if you don't pass the variables. I can't really see any reason why one would want to `go install` a binary that they can't run on the local machine. wdyt?

Reply:
Yeah, that's a fair point. LGTM, I'll merge shortly!